packages/client/src/index.test.ts

```diff
@@ -146,13 +157,14 @@ describe('LogtoClient', () => {
         storage,
       });
       await logto.loginWithRedirect(REDIRECT_URI, jest.fn());
-      expect(storage.getItem('LOGTO_SESSION_MANAGER')).toHaveProperty('redirectUri', REDIRECT_URI);
-      expect(storage.getItem('LOGTO_SESSION_MANAGER')).toHaveProperty('codeVerifier');
+      const sessionPayload = storage.getItem('LOGTO_SESSION_MANAGER');
+      expect(sessionPayload).toHaveProperty('redirectUri', REDIRECT_URI);
+      expect(sessionPayload).toHaveProperty('codeVerifier');
```

Comment:
`state` needed? maybe introduce `Session` type in this test will better

Reply:
TBD @simeng-li @wangsijie @xiaoyijun 

1. `state` REQUIRED

    Even though RFC just recommend `state`, login-flow require `state` for better security.

2. DO NOT introduce Session
    
    At first I used import `SessionManager` type to get them.
    But Simeng recommended me to use `storage` to get directly.
    I also think that less dependency is better, so do not introduce session.

Reply:
1. you can test `state` existency
2. It's ok

Reply:
Fixed.